CONTRIBUTING.md

```diff
@@ -73,6 +73,9 @@ versioning guidelines:
   `envoy.features.enable_all_deprecated_features` is set to true. Finally, following the deprecation
   of the API major version where the field was first marked deprecated, the entire implementation
   code will be removed from the Envoy implementation.
+* If the runtime flag `envoy.features.allow_deprecated_features` is disabled,
```

Comment:
I think the name is a bit confusing given we already have `envoy.features.enable_all_deprecates_features`. How about `envoy.features.fail_on_any_deprecated_feature` with reversed logic?

Reply:
runtime flag -> runtime key here and below, please